### ISSUE TITLE
package copy

### ISSUE THREAD

**Issue by brasizza** [warith313/sunmi_printer_plugin#1]
Is this a copy of my package ?

https://pub.dev/packages/sunmi_printer_plus

**Comment by brasizza** [comment]
@warith313

**Comment by warith313** [comment]
Honestly, I don’t remember exactly, but I did make some modifications and customizations to certain packages to fit my needs, and yours might be one of them. I apologize if this causes you any inconvenience.

**Comment by brasizza** [comment]
> Honestly, I don’t remember exactly, but I did make some modifications and customizations to certain packages to fit my needs, and yours might be one of them. I apologize if this causes you any inconvenience.

No problem at all, but many packages with same solution is bad to the final user.
I changed my whole package to use direct sdk from sunmi, and if you want im more then happy to discuss your modifications or customization to a PR in the project

**Comment by brasizza** [comment]
> Honestly, I don’t remember exactly, but I did make some modifications and customizations to certain packages to fit my needs, and yours might be one of them. I apologize if this causes you any inconvenience.

Looking at your commits you didn't change nothing, just the yaml description, but that's ok, the beauty of the flutter is the open source movement

**Comment by warith313** [comment]
I completely agree that having multiple packages offering the same solution can sometimes create unnecessary complexity for end users. That’s why I appreciate your efforts to streamline things by using the direct SDK from Sunmi—it sounds like a solid approach.

Regarding the commits, you’re right that my changes were minimal. I didn’t need to make deeper modifications at that time, but I’m always open to collaborating and contributing if there’s something valuable we can build.
